.github/workflows/minimum.yml

```diff
@@ -31,5 +31,6 @@ jobs:
       run: |
           python -m pip install --upgrade pip
           python -m pip install invoke .[test]
+          python -m pip install "scikit-learn<1.8.0"
```

Comment:
why is this not a requirement?

Reply:
Scikit-learn is an optional dependency for the tutorials. 1.8.0 recently released and no longer supports older versions of scipy and numpy which is why the minimum tests were originally failing.